package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@aboutbits/material-design-icons",
+  "version": "0.1.0",
+  "description": "Material design icon components for React",
+  "main": "dist/index.ts",
+  "scripts": {
+    "build": "npm run bootstrapping",
+    "bootstrapping": "node src/bootstrapper.js",
+    "clean": "rm -rf dist/",
+    "lint": "eslint --ext js,ts,tsx src",
+    "lint:fix": "npm run lint -- --fix",
+    "test": "jest --config jestconfig.json --passWithNoTests",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm run test && npm run lint",
+    "version": "npm run test && npm run lint && git add -A src",
+    "postversion": "git push && git push --tags"
+  },
+  "keywords": [
+    "material design icons",
```

Comment:
I would split "material design" and "icons" up into two different strings to optimize the search results.